### DISCUSSION TITLE
Grid Renewables

### DISCUSSION THREAD

**Discussion by jonbartlett** [amberelectric/public-api#60] [General]
Hi,

Just wondering where the numbers for share of renewables in the grid come from? Yesterday there were some big spikes which I couldn't see when looking at OpenNEM. What would cause these spikes (blue line)?

<img width="342" alt="image" src="https://user-images.githubusercontent.com/119513/129498849-edc7e971-6243-4dfc-a4d7-6d58eaaf9749.png">

Also a crazy day for the consumption price (red line) being very much negative for a while there (very windy and sunny in Victoria so assume this explains it?).

Thanks.

**Comment by madpilot** [comment]
Hey Jon,

It's a combination of the AEMO predicted semi-scheduled generation (ie solar and wind farm) and rooftop solar numbers.

I'm looking at my own graph for that and it looks like the spikes are at the end of the period (ie from :25-:30 and :55-:00), so my guess would be the prediction engine was under predicting either the generation (most likely) or rooftop numbers, and when the final numbers came in there was a massive jump.

And you are right about the windy and sunny day yesterday was causing some decent negative prices for much of the day.

**Comment by jonbartlett** [comment]
Thanks @madpilot 

So is this metric calculated by Amber? Or come direct from AEMO? and that it is estimated for the current period then adjusted as the 5 minute actuals come in?

Yesterday there were even wilder swings in the number (I don't think VIC has got to 100% renewables yet).

![image](https://user-images.githubusercontent.com/119513/130398790-29adf983-9326-4d4c-aaea-6430e1494cfa.png)

**Reply by madpilot** [reply]
We are just about to kick off the work around 5-minute pricing. I'll bring this up with the team as a talking point. That's my round about way of saying "I don't know, yet".

**Comment by jonbartlett** [comment]
The reason I ask is that I would like to use the renewables % as a metric to drive decisions around household energy usage. The spikes and accuracy of this info might make this problematic.

**Reply by madpilot** [reply]
So, I've found the reason: The rooftop renewables are only calculated for the 00th and :30th minutes, but our current price takes into account 5-minute prices. When we mash the two sets of numbers together, the rooftop only kicks in on the :00th and :30th minute.

This will be fixed as part of our 5-minute pricing work.

**Reply by jonbartlett** [reply]
Thanks for the update. Makes sense.

When is the 5 minute pricing going to be enabled?

**Reply by madpilot** [reply]
Tonight! But it will make basically no difference (other than a slightly different current price due to a change in our averaging algorithm) - no retail customers will have 30-minute usage enabled until the staged roll out starting January 2022.

I'm writing an FAQ which I'll post tomorrow after the content team finish with it.

**Reply by madpilot** [reply]
There was an issue with our renewable percentages which we are fixing at the moment. On the down side, it's currently not consistently showing rooftop solar, on the plus side when it gets fixed on Monday (just doing final testing) it won't have the spikes you are seeing anymore.
